config/core/test-configs.yaml

```diff
@@ -2443,6 +2456,7 @@ test_configs:
       - cros-ec
       - igt-gpu-panfrost
       - igt-kms-rockchip
+      - kselftest-filesystems
```

Comment:
I don't think it's a good idea to add tests to `veyron-jaq` as it's an old platform with only 2 online. Maybe just adding this one will be fine, we could do that and see if they can deal with the load. But even if this works, I wouldn't be surprised if we were actually pretty close to the capacity limit.